README.md

```diff
@@ -61,6 +61,24 @@ var env = multiplex.engines.nunjucks;
 env.addGlobal('key', 'value');
 ```
 
+## Cross-engine Embedding
+
+### Nunjucks
+
+To embed a template, use an `embed` filter in the parent template. Remember to pass it through `|safe` so it's not escaped.
```

Comment:
I don't know if we can.  If other people are making components, we don't know what their setting is.